README.md

```diff
@@ -14,6 +14,16 @@ In addition, this opens the yaml file to templatization and customization with e
 go build
 ```
 
+## Releases
+
+Users should use the `x.X` releases for stable use cases (eg 0.7).
+
+Breaking changes may occur between `x.X` releases (eg 0.7 and 0.8), and will be documented in the changelog.
+
+- Pushes to the [`develop`](https://github.com/NYTimes/drone-gke/tree/develop) branch will update the Docker Hub release tagged `develop`.
+- Pushes to the [`master`](https://github.com/NYTimes/drone-gke/tree/master) branch will update the Docker Hub release tagged `latest` and `x.X` (eg 0.7).
+- Tags to the [`master`](https://github.com/NYTimes/drone-gke/tree/master) branch will create the Docker Hub release with the tag value (eg 0.7.1).
+
```

Comment:
Thanks for writing this up, but it still doesn't explain what the workflow is for this repo. Or what image tags are usable on which drones. E.g. "latest" doesn't actually work on the latest version of drone.

It also appears that at least several teams have been told to use the develop tag, which they're using for production applications. It's counterintuitive that updating the "develop" branch breaks people's production deployments. But there is also no 0.7 tag to use instead: https://hub.docker.com/r/nytimes/drone-gke/tags/

The presence of a develop branch indicates git-flow, but there are commits in master that are not in develop.

Reply:
Ok, I will update this to talk about the git workflow.
As far as I can tell, commits to `master` were merged into `develop`.

What we can do about the `docker` develop tag:
- merge this
- tag to update the 0.4 release
- start a 0.4.x branch from master to keep the code for 0.4
- merge `develop` to `master`
- create a 0.7 release from `develop` as is for people using the `develop` docker tag to pin against